Lib/test/pickletester.py

```diff
@@ -2805,12 +2814,22 @@ class Nested(str):
             (Subclass.Nested("sweet").count, ("e",)),
             (Subclass.Nested.count, (Subclass.Nested("sweet"), "e")),
         )
-        for proto in range(pickle.HIGHEST_PROTOCOL + 1):
+        #for proto in range(pickle.HIGHEST_PROTOCOL + 1):
```

Comment:
?

Reply:
Good catch! This is a debugging remnants.